en/docs/InstallAndSetup/DeployingWSO2APIManager/deploying-api-manager-with-kubernetes-or-openshift-resources.md

```diff
@@ -1,187 +1,91 @@
 # Deploying API Manager with Kubernetes or OpenShift Resources
 
```

Comment:
Let's remove Openshift title here. Without testing this in Openshift, we can't say this will work. There are some other changes required to get this working in Openshift. IE team will release Openshift related artifacts in the future.

Reply:
ok will remove it